vision/cloud-client/document_text/doctext.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Outlines document text given an image.
+
+Example:
+    python doctext.py resources/cropme.jpg
+"""
+# [START full_tutorial]
+# [START imports]
+import argparse
+from enum import Enum
+import io
+
+from google.cloud import vision
+from PIL import Image, ImageDraw
+# [END imports]
+
+
+class FeatureType(Enum):
+    PAGE = 1
+    BLOCK = 2
+    PARA = 3
+    WORD = 4
+    SYMBOL = 5
+
+
+def draw_boxes(im, blocks, color, width):
+    """Draw a border around the image using the hints in the vector list."""
+    # [START draw_blocks]
+    draw = ImageDraw.Draw(im)
+
+    for block in blocks:
```

Comment:
I meant to ask this in the other sample, but why not use http://pillow.readthedocs.io/en/3.1.x/reference/ImageDraw.html#PIL.ImageDraw.PIL.ImageDraw.Draw.polygon ?